eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -9,4 +9,4 @@ steps:
       workingDirectory: $(Build.SourcesDirectory)/${{ parameters.Directory }}
       filePath: eng/common/scripts/Verify-Links.ps1 
       arguments: >
-        -urls $(dir -r -i *.md) -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" -recursive $false
+        -urls $(dir -r -i *.md) -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" -recursive:$false -ignoreLinksFile "$(Build.SourcesDirectory)/eng/ignore-links.txt"
```

Comment:
We should make the ignore file a parameter to this template. You can default it to the value you have but we should enable folks to pass in a different path if needed.